README.md

```diff
@@ -113,14 +113,12 @@ Jekyll Special elements
 
 
 Translations
-=============================
-
-The base content is in english under the "_langs/en" directory. Translations can be added following the same structure with either a two letter or composite language code: e.g. "es", "pt-br".
-
-To test translations set the lang variabe in the config/local.yml. For example, to set the mode to French:
-
-    lang: "fr"
+============
 
-To ensure the content is built when being released onto developers.google.com, ensure that the langs_available list in config/devsite.yml is updated to include any new language directories being added. For example, to add French:
+All article sources are in `src/_langs`. The base content is in english under the "en" directory. Translations can be added following the same structure with either a two letter or composite language code: e.g. "es", "pt-br".
```

Comment:
Capitalize 'english'.

'Translations can be added..' If you have time, change to active voice. 'Add translations by following...'